terraform/modules/helm/velero/variables.tf

```diff
@@ -104,6 +104,7 @@ variable "velero_backup_bucket_region" {
 variable "velero_sa_iam_role_name" {
   type        = string
   description = "Service Account IAM Role Name"
+  default     = ""
```

Comment:
Why do we need an emptry String as default?

Reply:
needed regarding gcp changes